docs/book/src/plugins/go-v3-plugin.md

```diff
@@ -0,0 +1,22 @@
+# go/v3
+
+This is the default plugin that is used by Kubebuilder to construct sets of [controllers](https://github.com/kubernetes-sigs/controller-runtime). It basically scaffolds all the boilerplate code required to create and design controllers. Check the following [example](https://github.com/kubernetes-sigs/kubebuilder/tree/master/testdata/project-v3) to view the files created or updated by this plugin when used with the supported subcommands. You can also follow the [Building CronJob](cronjob-tutorial.md) tutorial to get a step-by-step explanation of how the plugin works with different subcommands.
```

Comment:
```suggestion
Kubebuilder tool will scaffold the go/v3 plugin by default. This plugin is a composition of the plugins ` kustomize.common.kubebuilder.io/v1` and `base.go.kubebuilder.io/v3`. By using you can scaffold the default project which is a helper to construct sets of [controllers](https://github.com/kubernetes-sigs/controller-runtime). 

It basically scaffolds all the boilerplate code required to create and design controllers. Note that by following the [quickstart][quickstart] you will be using this plugin. 

<aside class="note">
<h1>Examples</h1>

Samples are provided under the [testdata][testdata] directory of the Kubebuilder project. You can check the go/v3 content by looking at all projects which are named with `v3`. Check the project-v3 to see the common based without the additional options. 

</aside>

Reply:
We also need add the alias [link]: link at the bottom 
See the doc added for the declarative plugin already.